benchexec/tablegenerator/react-table/scripts/start.js

```diff
@@ -40,6 +41,18 @@ const createDevServerConfig = require("../config/webpackDevServer.config");
 const useYarn = fs.existsSync(paths.yarnLockFile);
 const isInteractive = process.stdout.isTTY;
 
+const dataParam = process.argv[2];
+
+if (dataParam) {
+  const content = fs.readFileSync(path.resolve(__dirname, "../", dataParam));
+  fs.writeFileSync(path.resolve("./src/data/custom-data.json"), content);
```

Comment:
Copying the file is a nice and simple solution, I like that.

I guess we could use [copyFileSync](https://nodejs.org/api/fs.html#fs_fs_copyfilesync_src_dest_flags)?

Reply:
Nice catch, will refactor this part